exercises/sam_jenkins/ex02_oo_basics/src/Circle.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Shapes.h"
+#include "Circle.h"
+#include <math.h>
+
+	Circle::Circle() :radius(1)
+	{
+		type = "Circle";
+		sides = 1;
```

Comment:
Why not initialise these variables like radius? Then you should be able to make them `const`

Reply:
I did originally but changed it due to running into problems with the linker